webpack.config.js

```diff
@@ -17,7 +17,8 @@ const plugins = [
   new webpack.DefinePlugin({
     // <-- key to reducing React's size
     'process.env': {
-      NODE_ENV: JSON.stringify(process.env.NODE_ENV || 'dev'),
+      // NODE_ENV: JSON.stringify(process.env.NODE_ENV || 'dev'),
```

Comment:
Thanks for catching that. I was commenting it out for local use to avoid a known issue of conflicting env variable. This is because Webpack 4+ automatically injects `process.env.NODE_ENV` and makes it available to the bundle output based on `mode` option: https://webpack.js.org/configuration/mode/.

In practice, commenting out the line shouldn't impact the production bundle. However, to be extra careful, I'll push a commit to revert it as a short-term measure.

That said, this needs a more fundamental solution re: how the portal codebase relies on non-standard values for `NODE_ENV`: i.e. using `"dev"` instead of `"development"` for development and `"auto"` for... something I'm not sure about. I'll create a ticket on this and make a PR to address it.